source/sass/components/chapter-page.scss

```diff
@@ -0,0 +1,151 @@
+// @TODO: This component needs design polish
```

Comment:
This will need a reference pass against the already established styleguide, because making things a _little_ bit different from the comps if it means reusing styles we already formally declared in our styleguide is definitely preferable to independent styling.

Reply:
The scss will need a looking at from Mavis probably. Theres probably font size functions laying around, for instance, that I should be using instead of writing each one manually. 

@mmmavis could you take a look please?